libs/ui/src/components/NeoIcon/NeoIcon.vue

```diff
@@ -2,7 +2,7 @@
   <o-icon
     :pack="pack || 'fas'"
     :icon="icon"
-    :size="size || 'small'"
+    :size="size || ''"
```

Comment:
```suggestion
    :size="size"
```

Reply:
default size isn't in `'small' | 'medium' | 'large'`

Reply:
> default size isn't in 'small' | 'medium' | 'large'

If so, we should define the correct type for the size props in line 14.  Replacing 'small' with an empty string is not a good solution.

> so would it work if provide empty?
If it does not work, we should provide a proper default value for it.